trade-finance-manager-ui/component-tests/accessibility/accessibility.component-test.js

```diff
@@ -0,0 +1,71 @@
+const pageRenderer = require('../pageRenderer');
+
+const page = '../templates/accessibility-statement.njk';
+
+const render = pageRenderer(page);
+
+describe(page, () => {
+  let wrapper;
+  const params = {
+    dealId: '1234',
```

Comment:
Most likely this id parameter is not required.

Reply:
deaId is removed